### ISSUE TITLE
Entity types pages don't show all the entities

### ISSUE THREAD

**Issue by ziodave** [insideout10/wordlift-plugin#755]
Please describe the issue here:

https://freeyork.org/wl_entity_type/organization/
wl entity organisation doesn’t return any results

WL entity creative work returns only 3 results whereas it has 69
https://freeyork.org/wl_entity_type/creative-work/

Please provide additional data about your WordPress install:
*[ ] WordPress version: 4.9.1
*[ ] WordLift plugin version: 3.16.5

**Comment by Stoyan0v** [comment]
The issue is not related to WordLift plugin, but the `Bimper` theme the site uses. There is a plugin called Snax which has two custom post types. The child theme uses the following hook to add that post type to the mix of wp_query post types:
`add_action( 'pre_get_posts', 'bimber_snax_add_cpt_to_queries' );`

The `bimber_snax_add_cpt_to_queries` function gets the current post types from the query and adds snax post types:

```
$post_type = $query->get( 'post_type' );

// Normalize.
$post_type = ( '' === $post_type ) ? array( 'post' ) : (array) $post_type;

// Skip if query is not for "post" type.
if ( ! in_array( 'post', $post_type, true ) ) {
	return;
}

$post_type[] = snax_get_quiz_post_type();
$post_type[] = snax_get_poll_post_type();

$query->set( 'post_type', $post_type );
```

At the 

Everything seems ok, but for taxonomy page the post type is usually empty and it's added later by getting the registered custom post types for the taxonomy ([link](https://github.com/WordPress/WordPress/blob/master/wp-includes/class-wp-query.php#L2067)).

Some of the entity type pages are displaying posts, because there are posts under as "Creative Work" type for example.

To sum up: The issue is related to the query "POST IN" clause which is modified by the theme ans most exactly by `bimber_snax_add_cpt_to_queries` function.
